lib/src/face_service_rest_client.dart

```diff
@@ -54,6 +54,7 @@ class FaceServiceClient {
     bool returnFaceId = true,
     bool returnFaceLandmarks = false,
     bool returnRecognitionModel = false,
+    String detectionModel = "detection_03",
```

Comment:
It is an unused variable

Reply:
As I understand, this parameter is required for smoother search of Similar Faces and LargeFaceLists